packages/interaction-kit/src/cli/dev.ts

```diff
@@ -0,0 +1,172 @@
+import ngrok from "ngrok";
+import chokidar from "chokidar";
+import arg from "arg";
+import * as API from "../api";
+import {
+	getApplicationEntrypoint,
+	getGuildApplicationCommandChanges,
+} from "../scripts";
+import Application from "../application";
+import { Snowflake, ApplicationCommand } from "../definitions";
+
+const CONFIG_FILES = [".env"];
+const BOT_FILES = ["package.json", "src/**/*"];
+
+/* eslint-disable */
+// @ts-expect-error TS Dumb
```

Comment:
More description on why dumb?

Reply:
Doesn't exist yet, but yeah will fix that